### ISSUE TITLE
Notice: Undefined property: OpenTok\StreamList::$items

### ISSUE THREAD

**Issue by rbaaboud** [opentok/OpenTok-PHP-SDK#237]
Hi,

When I use `OpenTok\StreamList::getItems()` method, I have the following Notice: 

```
Notice: Undefined property: OpenTok\StreamList::$items in /var/www/vendor/opentok/opentok/src/OpenTok/StreamList.php on line 33
```

**Comment by rbaaboud** [comment]
Hi,
Up :)
Pull request : #238 
Thanks :)

**Comment by msach22** [comment]
@rbaaboud Hmm, I just ran the hello world sample application with this and I was able to get it working. I believe this issue should only happen when you call `getItems` when there are no active clients connected to the session because the response would then be empty. 

Is this when this happens for you?

**Comment by Hypnopompia** [comment]
@msach22 That's what's happening for me.

**Comment by msach22** [comment]
@Hypnopompia I'll be happy to accept the PR, but please note that you should only call this method when there are active streams in the session.

**Comment by Hypnopompia** [comment]
Normally, that's the case. However there are certain race conditions where we think there are open streams, then do an API to get a list of streams, but they've been closed in the meantime. That's when we're running into this issue.

**Comment by msach22** [comment]
Thanks for sharing - I appreciate you all finding the issue and submitting the PR.

**Comment by rbaaboud** [comment]
@msach22 sorry for the delay

Thanks for your merge.

**Comment by rbaaboud** [comment]
@msach22 Did you plan a new release with at least this fix?

Thanks

**Comment by msach22** [comment]
This has been released with v4.4.1
